src/scripts/interface/gameManager.ts

```
@@ -0,0 +1,27 @@
+      document.body.classList.remove('white-player');
+      document.body.classList.add('red-player');
+    } else {
+      document.body.classList.add('white-player');
+      document.body.classList.remove('red-player');
+    }
+  }
+
+  public static registerEventHandler(game: GameInfo) {
+    game.registerEventHandler(GameManager.handleGameEvent);
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)